src/Upgrade/Infrastructure/Configuration/ConfigurationProvider.php

```diff
@@ -56,6 +74,9 @@ public function getSourceCodeProvider(): string
     }
 
     /**
+     * Specification:
+     * - The method define pattern for branch that will be created during upgrade process.
```

Comment:
Defines pattern for branch that will be created during upgrade process.

Reply:
++